src/main/java/ua/lviv/iot/novaPoshtaAPI/Util.java

```diff
@@ -0,0 +1,25 @@
+package ua.lviv.iot.novaPoshtaAPI;
+
+import java.time.LocalDate;
+
+public class Util {
```

Comment:
this class is not used in the code

Reply:
it is used, but I'll add one more method with next commit